### PR TITLE
Catch credentials TypeError

### PR DESCRIPTION
**Pull request by HalleyInteractive**
Perhaps there's a better way to catch both (Or others?)

### CODE REVIEW

**Review by caiotomazelli — CHANGES_REQUESTED** [review]

File: dags/sources/bigquery.py [hunk 1]
```diff
@@ -41,6 +41,9 @@ def __init__(self, config: Dict[str, Any]):
     except ValueError:
       # json.loads fails if credentials are not a valid JSON object
       config["credentials"] = None
+    except TypeError:
```
[the remark this answers is not in the document]
Comment (HalleyInteractive) [reply dags/sources/bigquery.py:44]: Done

File: dags/sources/bigquery.py [hunk 2]
```diff
@@ -38,7 +38,10 @@ class Source:
   def __init__(self, config: Dict[str, Any]):
     try:
       json.loads(config.get("credentials"))
-    except ValueError:
+    except (ValueError, TypeError):
+      # json.loads fails if credentials are not a valid JSON object
+      config["credentials"] = None
+    except TypeError:
```
Comment (caiotomazelli) [inline dags/sources/bigquery.py:44]: I think this line is residual

**Review by caiotomazelli — CHANGES_REQUESTED** [review]

File: dags/sources/bigquery.py (hunk 2 above)
Comment (caiotomazelli) [reply dags/sources/bigquery.py:44]: line 44

Comment (HalleyInteractive) [reply dags/sources/bigquery.py]: Thanks, not sure how it got back in there :) This is now removed with the latest commit.

**Review by caiotomazelli — APPROVED** [review]

**Review by caiotomazelli — APPROVED** [review]

**Pull request merged**